equinix/README.md

```diff
@@ -47,7 +47,13 @@ The `harvester.install.config_url=https://metadata.platformequinix.com/userdata`
       password: p@ssword  # replace with a your password
     install:
       mode: create
-      mgmt_interface: eth0
+      networks:
+        harvester-mgmt: # The management bond name. This is mandatory.
+          bond_option:
+            mode: balance-tlb
+          interfaces:
+          - name: eth0
```

Comment:
Do we also need to specify `default_route: true` here to establish the default route? If not, is default route assumed if absent?

Reply:
`harvester-mgmt` becomes the default route if none of networks there has a `default_route: true`. It is great to add `default_route` for the purpose of an example. WDYT?

Reply:
Yes, I think we can use that to illustrate the choice for default route.